Gemfile.lock

```diff
@@ -86,14 +86,14 @@ GEM
     backport (1.2.0)
     base64 (0.2.0)
     benchmark (0.3.0)
-    better_html (2.0.2)
+    better_html (2.1.1)
```

Comment:
Akoze prezijeme to, ale bump gemov skusme vynechavat z PR pokial nie je nutny.

Reply:
Plan bol na konci to vyhodit. Su to len development zavislosti. Ale mozem aj nechat

Reply:
Uz vyhodime?